trianglePattern/TrianglePatternProgram.java

```diff
@@ -0,0 +1,23 @@
+package trianglePattern;
+
+public class TrianglePatternProgram {
+    public static void main(String[] args) {
+        int number = 7;
+        trianglePattern(number);
+    }
+
+    public static void trianglePattern(int number) {
+        for (int rows = 0; rows <= number - 1; rows++) {
+            for (int column = 0; column <= rows; column++) {
+                System.out.print("*" + " ");
+            }
+            System.out.println("");
+        }
+        for (int rows = number - 1; rows >= 0; rows--) {
```

Comment:
in for loop always try to use different variable name

Reply:
Changed variable names